build-plugins/src/main/kotlin/aws/sdk/kotlin/gradle/kmp/ConfigureTargets.kt

```diff
@@ -156,6 +160,16 @@ fun Project.configureJvm() {
     }
 }
 
+fun Project.configureLinux() {
+    kotlin {
+        linuxX64 {
+            tasks.named("linuxX64Test") {
+                enabled = false
+            }
+        }
+    }
+}
+
```

Comment:
**Nit**: Probably leave a TODO here for removing/re-enabling this once we _really_ support Linux.